### PR TITLE
Removed un used code related to Appdomain

### PR DESCRIPTION
**Pull request by acesiddhu**
## Description
Removed in used code related to Appdomain in Testplaform

### CODE REVIEW

**Review by abhishkk — APPROVED** [review]

File: src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyHelper.cs [hunk 1]
```diff
@@ -1,345 +0,0 @@
...
-        private static Version defaultVersion = new Version();
-        private static Version version45 = new Version("4.5");
-
-        /// <summary>
-        /// Checks whether the source assembly directly references given assembly or not.
-        /// Only assembly name and publickey token are match. Version is ignored for matching.
-        /// Returns null if not able to check if source references assembly.
-        /// </summary>
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Design", "CA1031:DoNotCatchGeneralExceptionTypes")]
-        public static bool? DoesReferencesAssembly(string source, AssemblyName referenceAssembly)
```
Comment (smadala) [inline src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyHelper.cs]: This is public API, Can be used by old adapters. [Example](https://github.com/Microsoft/testfx/blob/abd5ea3585657914490c83d5d2b4c9e732640a89/src/Adapter/PlatformServices.Desktop/DesktopTestSource.cs#L46)

File: src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyHelper.cs [hunk 2]
```diff
@@ -1,345 +0,0 @@
...
-        }
-
-        /// <summary>
-        /// Finds platform and .Net framework version for a given test container.
-        /// If there is an error while infering this information, defaults (AnyCPU, None) are returned
-        /// for faulting container.
-        /// </summary>
-        /// <param name="testSource"></param>
-        /// <returns></returns>
-        public static KeyValuePair<Architecture, FrameworkVersion> GetFrameworkVersionAndArchitectureForSource(string testSource)
```
Comment (smadala) [inline src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyHelper.cs]: I believe this was used by testwindow. Not sure it relevant anymore.

File: src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyHelper.cs [hunk 3]
```diff
@@ -1,345 +0,0 @@
...
-        }
-
-
-        /// <summary>
-        /// Returns the full name (AssemblyName.FullName) of the referenced assemblies by the assembly on the specified path.
-        /// 
-        /// Returns null on failure and an empty array if there is no reference in the project. 
-        /// </summary>
-        /// <param name="source">Full path to the assembly to get dependencies for.</param>
-        public static string[] GetReferencedAssemblies(string source)
```
Comment (smadala) [inline src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyHelper.cs]: same as `DoesReferencesAssembly`.

**Review by smadala — CHANGES_REQUESTED** [review]

File: src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyHelper.cs (hunk 2 above)
Comment (acesiddhu) [reply src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyHelper.cs]: why is testwindow depedant on objectmodel?

Comment (mayankbansal018) [reply src/Microsoft.TestPlatform.ObjectModel/Utilities/AssemblyHelper.cs]: [AssemblyMetadataProvider](https://github.com/Microsoft/vstest/blob/master/src/vstest.console/CommandLine/AssemblyMetadataProvider.cs) does the same work as this method?

**Pull request closed without merging**